package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "poweruse",
```

Comment:
Hvorfor ændre name til `html` ?

Reply:
ahh det her PR er fixed i https://github.com/Tvup/poweruse/pull/114 😄

Reply:
Altså jeg fuckede op - jeg pushede til den her branch, og det er jo allerede passé.
Så ville jeg prøve at redde det ved at flytte HEAD bagud og rebase ......
Altså nogle gange kan man også bare gøre det svært for sig selv.
Jeg lavede den her draft for at se, hvad der egentlig var sket :joy: